packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -113,23 +105,52 @@ export default class OperatorModeOverlays extends Component<Signature> {
     </style>
   </template>
 
+  @tracked isHoverOnCard = new TrackedWeakMap<RenderedLinksToCard, boolean>();
+  isEventsRegistered = new TrackedWeakMap<RenderedLinksToCard, boolean>();
+
   offset = {
     name: 'offset',
     fn: (state: MiddlewareState) => {
       let { elements, rects } = state;
       let { floating, reference } = elements;
       let { width, height } = reference.getBoundingClientRect();
-      
+
       floating.style.width = width + 'px';
       floating.style.height = height + 'px';
-
+      floating.style.position = 'absolute';
       return {
         x: rects.reference.x,
         y: rects.reference.y,
       };
     },
   };
 
+  // Since we disable pointer events of this overlay component,
```

Comment:
Was hoping to see a more thorough comment on why we disable them. I think an explanation about the actual problem is needed here. The problem where the card is vertically long and thus needs scrolling, but scrolling unexpectedly stops when the cursor starts hovering overlaid actions on top of cards

Reply:
Updated.